tasks/uptime-check.js

```diff
@@ -0,0 +1,66 @@
+// usage:
+//
+//   PROJECT_JSON=path/to/projects.json node tasks/uptime-check.js
+
+'use strict';
+
+const fs = require('fs');
+const request = require('request');
+
+
+const getProjectJson = () => {
+  const jsonPath = process.env.PROJECT_JSON || './projects-json/projects.json';
+  const content = fs.readFileSync(jsonPath);
+  return JSON.parse(content);
+};
+
+const getUrl = (linkObj) => {
+  if (typeof linkObj == 'string') {
+    return linkObj;
+  } else {
+    return linkObj.url;
+  }
+};
+
+const headReq = (uri, callback) => {
+  request.head(uri, callback);
+};
+
+const isSuccess = (res) => {
+  return (res.statusCode >= 200 && res.statusCode < 300) || res.statusCode === 403;
+};
+
+const checkIfUp = (uri, callback) => {
+  headReq(uri, (err, res, body) => {
+    if (err) {
+      callback(err)
+    } else {
+      callback(null, isSuccess(res))
+    }
```

Comment:
I would ditch the if/else expression, for a simple one-liner:

```javascript
   callback(err, isSuccess(res))
```

As long as isSuccess won't throw. You might want to add a `!!res` check to `isSuccess` just so it won't throw if `res` is undefined.